viite-UI/src/view/FormCommon.js

```diff
@@ -35,19 +35,19 @@
     const projectButtonsRecalculateAndChanges = function () {
       return  '<button class="recalculate btn btn-block btn-recalculate">Päivitä etäisyyslukemat</button>' +
           '<button class="show-changes btn btn-block btn-show-changes">Avaa projektin yhteenvetotaulukko</button>' +
-        '<button disabled id ="send-button" class="send btn btn-block btn-send">Hyväksy tieosoitemuutokset</button>';
+        '<button disabled title="Hyväksy yhteenvedon jälkeen" id="send-button" class="send btn btn-block btn-send">Hyväksy tieosoitemuutokset</button>';
```

Comment:
Mieluummin titlen sijaan tooltip - ellei tooltipin käytölle ole jotain estettä?

Reply:
miksi? Jos on olemassa html atribuutti joka toimii nappulan tapauksessa juuri niin kuin tooltip niin miksi lähteä väsäämään css ja span härpäkkää että saa tismalleen saman lopputuoksen? 😄
https://stackoverflow.com/questions/2238239/tooltips-for-button-elements

Reply:
"title
Contains a text representing advisory information related to the element it belongs to. Such information can typically, but not necessarily, be presented to the user as a tooltip."  
https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-title

Reply:
Ok, hyvin perusteltu, vedän takas! :)